scripts/brew-install.sh

```diff
@@ -10,5 +10,5 @@ export HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1
 export HOMEBREW_NO_AUTO_UPDATE=1
 export HOMEBREW_NO_ANALYTICS=1
 
-brew install node@18
-brew link --overwrite node@18
+brew reinstall node@20
+brew link --overwrite node@20
```

Comment:
The only thing that actually requires node 18 is `pkg`. So everything should still work on node 20. And then we will eventually update to node 20. But since I just started using nodejs, maybe we just all swap to using node 18?

Reply:
I've set it to node 18 for now. That's specific to PK CLI due to pkg not supporting node 20 any time soon.